onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -303,7 +303,7 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
 #endif
 #ifdef ENABLE_NVTX_PROFILE
       profile::NvtxRangeCreator node_compute_range(
-          MakeString(node.OpType(), ".", node.Index(), "(", node.Name(), ")"), profile::Color::Blue);
```

Comment:
`Blue` has been used for a special pipeline operator (such as Send). If we keep using 'Blue` for other operators, it will be harder to visualize pipeline computation.